content/en/docs/kitex/Tutorials/cli-tool/kitexcall.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "CLI tool kitexcall"
```

Comment:
kitexcall: the CLI tool for sending RPC call in JSON format

Reply:
all fixed and checked